packages/foam-vscode/src/features/create-from-template.ts

```diff
@@ -11,54 +11,122 @@ import { FoamFeature } from '../types';
 import { TextEncoder } from 'util';
 import { focusNote } from '../utils';
 
-const templatesDir = `${workspace.workspaceFolders[0].uri.path}/.foam/templates`;
+const templatesDir = `${workspace.workspaceFolders[0].uri.fsPath}/.foam/templates`;
```

Comment:
why `fsPath` here instead of `path`? Generally speaking we should be very wary of using `fsPath` unless absolutely necessary

edit: Looking at this more closely I would actually have this as a `URI` (see my comment below about `URI` vs `path` vs `fsPath`)